deb-package/build

```diff
@@ -10,7 +10,7 @@ set -xeu
 
 # Set up package
 pkgname=libmxnet
-pkgversion_number=$(git describe --dirty |
+pkgversion_number=$(git describe --tags --dirty |
```

Comment:
Yes, we will only care about tags we make and using annotated tags can introduce problems as most our infra assumes tags are annotated.